docs/rst/user_manual/wan_configuration.rst

```diff
@@ -101,7 +101,7 @@ TLS
 ===
 
 |eddsrouter| also supports `TLS over TCP <https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/tcp/tls.html>`_,
-and its configuration can be set per participant for types Local Discovery Server and WAN. Following is a list of the
+and its configuration can be set per participant for types WAN Discovery Server and WAN. Following is a list of the
```

Comment:
(above in 4.1.1.1 External port)
Write explicitly the tag for external port between parenthesis (`external-port`).